RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py

```diff
@@ -67,20 +67,24 @@
 from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
 (pp_on_XeXe_2017 | pp_on_AA).toModify(offlinePrimaryVertices,
                TkFilterParameters = dict(
+                   algorithm="filterWithThreshold",
                    maxD0Significance = 2.0,
                    maxD0Error = 10.0, 
                    maxDzError = 10.0, 
                    minPixelLayersWithHits=3,
                    minPt = 0.7,
-                   trackQuality = "highPurity"
+                   trackQuality = "highPurity",
+                   numTracksThreshold = cms.int32(10),
+                   maxNumTracksThreshold = cms.int32(1000),
+                   minPtTight = cms.double(1.0)
```

Comment:
```suggestion
                   numTracksThreshold = 10,
                   maxNumTracksThreshold = 1000,
                   minPtTight = 1.0
```

Reply:
@jpata These parameters are not defined in the base PSet before era modification (e.g. L12-22 of this file), so in my tests this proposed change leads to errors at runtime.